### DISCUSSION TITLE
Pytorch Lightning Progress Bar Explained

### DISCUSSION THREAD

**Discussion by GeorgePearse** [Lightning-AI/pytorch-lightning#12322] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
I'm using batch sizes of 32 and a 1000 image dataset. I thought the fraction at the right of the progress bar woudl have the dataset size over batch size as the denominator, e.g. how many batches through the epoch are you. But the maths doesn't add up? 

32 * 383 = 12256

![image|658x89](upload://raCfvIkOgY6K6N4Yg6CBHmlIul2.png)

**Comment by akihironitta** [comment]
Hi @GeorgePearse, could you update the image? I can't see anything :eyes:

**Reply by GeorgePearse** [reply]
Whoops, not that image, but I can redo the maths for a different one.

batch size = 32 
training dataset = 1000

I'm really just trying to workout what the denominator means. Thanks a lot in advance!

![image](https://user-images.githubusercontent.com/47161914/158270205-6683bbe4-cc2e-46ba-b36c-9428295b6fdf.png)

**Reply by rohitgr7** [reply]
this is the main progress bar that shows the train batches + val batches
